js/mod.ts

```diff
@@ -2,16 +2,16 @@
 
 import { instantiate } from "./deno_doc_wasm.generated.js";
 import type { DocNode } from "./types.d.ts";
-import { load as defaultLoad } from "https://deno.land/x/deno_graph@0.64.1/loader.ts";
+import { createCache } from "https://deno.land/x/deno_cache@0.7.1/mod.ts";
```

Comment:
This now uses the cache from deno_cache, which will properly verify JSR checksums. I'm not sure this works on deploy though.